packages/node/jest.config.js

```diff
@@ -2,5 +2,8 @@ const rootConfig = require('../../jest.config')
 module.exports = {
     ...rootConfig,
     globalTeardown: './jest.teardown.js',
+    setupFilesAfterEnv: rootConfig.setupFilesAfterEnv.concat(
```

Comment:
Is this here by accident?

Reply:
No, needed for `toEqualBinary` in `createMessagingPluginTest.test.ts`.